From 98549a470361691537e7feadc2e519ea5aeb8320 Mon Sep 17 00:00:00 2001
From: Max Krummenacher <max.krummenacher@toradex.com>
Date: Sun, 6 May 2018 09:27:54 +0000
Subject: [PATCH] notification.cpp: include QStyleOption explicitely

5.11 Qt headers do not indirectly include QStyleOption resulting in compile
time errors like:
| notification.cpp:219:5: error: 'QStyleOption' was not declared in this scope

Upstream-Status: Submitted [https://github.com/lxqt/lxqt-notificationd/pull/79]

Signed-off-by: Max Krummenacher <max.krummenacher@toradex.com>
---
 src/notification.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/notification.cpp b/src/notification.cpp
index 4e1553d..333e974 100644
--- a/src/notification.cpp
+++ b/src/notification.cpp
@@ -35,6 +35,7 @@
 #include <KWindowSystem/KWindowSystem>
 #include <QMouseEvent>
 #include <QPushButton>
+#include <QStyleOption>
 
 #include "notification.h"
 #include "notificationwidgets.h"
-- 
2.13.6

